tests/packagedcode/data/podspec/SwiftLib/SwiftLib.podspec

```diff
@@ -0,0 +1,140 @@
+    #
+    #  Specify the authors of the library, with email addresses. Email addresses
+    #  of the authors are extracted from the SCM log. E.g. $ git log. CocoaPods also
+    #  accepts just a name if you'd rather not provide an email address.
+    #
+    #  Specify a social_media_url where others can refer to, for example a twitter
+    #  profile URL.
+    #
+  
+    spec.author             = { "alizainprasla" => "alizainprasla@gmail.com" }
+    # Or just: spec.author    = "jeantimex"
+    # spec.authors            = { "jeantimex" => "jean.timex@gmail.com" }
+    # spec.social_media_url   = "https://twitter.com/jeantimex"
+  
+    # ――― Platform Specifics ――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+    #
+    #  If this Pod runs only on iOS or OS X, then specify the platform and
+    #  the deployment target. You can optionally include the target after the platform.
+    #
+  
+    # spec.platform     = :ios
+    # spec.platform     = :ios, "5.0"
+  
+    #  When using multiple platforms
+    spec.ios.deployment_target = "12.1"
+    # spec.osx.deployment_target = "10.7"
+    # spec.watchos.deployment_target = "2.0"
+    # spec.tvos.deployment_target = "9.0"
+  
+    spec.swift_version = "4.2"
+  
+  
+    # ――― Source Location ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+    #
+    #  Specify the location from where the source should be retrieved.
+    #  Supports git, hg, bzr, svn and HTTP.
+    #
+  
+    spec.source       = { :git => "https://github.com/alizainprasla/swiftlib.git", :tag => "#{spec.version}" }
+  
+  
+    # ――― Source Code ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+    #
+    #  CocoaPods is smart about how it includes source code. For source files
+    #  giving a folder will include any swift, h, m, mm, c & cpp files.
+    #  For header files it will include any header in the folder.
+    #  Not including the public_header_files will make all headers public.
+    #
+  
+    spec.source_files  = "SwiftLib/**/*.{h,m,swift}"
+    # spec.exclude_files = "Classes/Exclude"
+  
+    # spec.public_header_files = "Classes/**/*.h"
+  
+  
+    # ――― Resources ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+    #
+    #  A list of resources included with the Pod. These are copied into the
+    #  target bundle with a build phase script. Anything else will be cleaned.
+    #  You can preserve files from being cleaned, please don't preserve
+    #  non-essential files like tests, examples and documentation.
+    #
+  
+    # spec.resource  = "icon.png"
+    # spec.resources = "Resources/*.png"
+  
+    # spec.preserve_paths = "FilesToSave", "MoreFilesToSave"
+  
+  
+    # ――― Project Linking ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+    #
+    #  Link your library with frameworks, or libraries. Libraries do not include
+    #  the lib prefix of their name.
+    #
+  
+    # spec.framework  = "SomeFramework"
```

Comment:
No sure, but do we want to track these as deps?

Reply:
frameworks are a list of system frameworks that the user’s target needs to link against. I don't think this comes under dependencies. What do you say??